packages/odyssey-react-mui/src/Pagination.tsx

```diff
@@ -73,44 +73,47 @@ const PaginationButtonContainer = styled("div")({
   },
 });
 
-export type DataTablePaginationProps = {
+export type PaginationProps = {
   pagination: {
     pageIndex: number;
     pageSize: number;
   };
```

Comment:
I don't think we need to use `material-react-table`'s nomenclature here. I'd rather these be separate props.

When passing them in, you can do `{...pagination}` in `DataTable`.

Reply:
Dome